VulkanHppGenerator.cpp

```diff
@@ -2373,7 +2434,7 @@ void writeFunctionBodyEnhanced(std::ostream & os, std::string const& indentation
   std::string returnName;
   if (commandData.returnParam != ~0)
   {
-    returnName = writeFunctionBodyEnhancedLocalReturnVariable(os, indentation, commandData, singular);
+    returnName = writeFunctionBodyEnhancedLocalReturnVariable(os, indentation, commandData, singular, isStructureChain);
```

Comment:
There's no need to change this function. The returnName for the structure changed is passed to it.